docs/xls-30d-amm/transaction-types/ammbid.md

```diff
@@ -126,9 +126,12 @@ Besides errors that can occur for all transactions, AMMBid transactions can resu
 
 | Error Code              | Description                                  |
 |:------------------------|:---------------------------------------------|
-| `temAMM_BAD_TOKENS`     | The specified `BidMin` or `BidMax` were not specified as the correct LP Tokens for this AMM. |
-| `tecAMM_FAILED_BID`     | This transaction could not win the auction, either because the sender does not hold enough LP Tokens to pay the necessary bid or because the price to win the auction was higher than the transaction's specified `BidMax` value. |
+| `tecAMM_BALANCE`        | The bid could not be applied, because the ***TODO: "invalid burn"?*** |
+| `tecAMM_EMPTY`          | The AMM has no assets in its pool. In this state, you can only delete the AMM or fund it with a new deposit. |
+| `tecAMM_FAILED`         | This transaction could not win the auction, either because the sender does not hold enough LP Tokens to pay the necessary bid or because the price to win the auction was higher than the transaction's specified `BidMax` value. |
 | `tecAMM_INVALID_TOKENS` | The sender of this transaction does not hold enough LP Tokens to meet the slot price. |
+| `tecINSUFFICIENT_PAYMENT` | The transaction could not refund the previous auction slot holder. ***TODO: why?*** |
```

Comment:
Also waiting from clarification from @gregtatcam on this error case.

Reply:
We figure out the auction slot price - payPrice. The previous owner of the auction slot is owed a refund, which is a fraction of the price that the slot was purchased for. We check that refund amount is less than the payPrice. If it is greater or equal to the payPrice then we return tecINSUFFICIENT_PAYMENT.

Reply:
Removed since this error case is impossible to reach normally.